### PR TITLE
Component Batching

### PR DESCRIPTION
**Pull request by xlanex6**
- add layout
- Make generic component for headings section
- Create Episode link on index page
- install dayJS
- Set LastEpisodes
- refacto and cleaning
- Basic ALL_EPISODES pages
- fix: Icons height on animate background
- DRY
- One page episode and lint
- Fix lint
- Query only MD not JSON on last episodes comp

### CODE REVIEW

File: content/podcasts/ecommerce-statique/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dsSlug: DS_004_ecommerce_statique
 author: Double slash
 duration: 3880
 season: 1
-episodeNumber: 4
+episodeNumber: 04
```
Comment (ipatate) [inline content/podcasts/ecommerce-statique/index.md:10]: you can use my futur plugin for that :) https://github.com/double-slash-podcast/website/blob/787ca7960587d71e957eabda1cacb59d8c55b088/plugins/formatTime.ts

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
+
+
+
+
+
+
+
+
+
```
Comment (ipatate) [inline readme.md:9]: space ++++

**Review by ipatate — APPROVED** [review]

File: components/PodcastList.vue [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<script setup>
+defineProps({
+  dark: {
+    type: Boolean,
+    default: false,
+  },
+});
+
+const plateforms = [
```
Comment (ipatate) [inline components/PodcastList.vue:9]: maybe we can define this in app.config.ts for centralize the infos ?

**Review by ipatate — APPROVED** [review]

**Review by ipatate — APPROVED** [review]
Where do you use day js ?

File: tailwind.config.js [hunk 4]
```diff
@@ -5,7 +5,9 @@ module.exports = {
     extend: {
       colors: {
         darkPurple: '#3C0C55',
-        yellowDs: '#FFE40F'
+        yellowDs: '#FFE40F',
+        purpleDS: '#9123CB',
```
Comment (ipatate) [inline tailwind.config.js:9]: DS or Ds

**Review by ipatate — APPROVED** [review]

File: tailwind.config.js (hunk 4 above)
Comment (xlanex6) [reply tailwind.config.js:9]: Ds  `everywhere` 
I will fix it 
`contentsy` is key

Comment (xlanex6) [reply tailwind.config.js]: @ipatate Can I merge ?

File: components/PodcastList.vue (hunk 3 above)
Comment (xlanex6) [reply components/PodcastList.vue:9]: yes it could be nice for SHOWS link
but it will be generate for EPISODES

**Pull request merged**